test/ERC721/ERC721Token.test.js

```diff
@@ -22,4 +23,5 @@ contract('ERC721Token should behave', function (accounts) {
   shouldBehaveLikeERC721BasicToken(accounts)
   shouldMintERC721Token(accounts)
   shouldBehaveLikeERC721Token(name, symbol, creator, accounts)
+  shouldBehaveLikeERC165(name, symbol, creator, accounts)
```

Comment:
Add this to the BasicToken.test.js and TokenProxy.test.js too

Reply:
Assume you mean ERC721BasicToken.test.js and TokenProxy.test.js?

Reply:
done